geometry.hpp

```diff
@@ -62,6 +62,10 @@ class Vector3d{
     return Vector3d(v[0]*d, v[1]*d, v[2]*d);
   }
 
+  double at( int i ) const {
+    return v[i];
+  }
+
```

Comment:
I don't know that I actually do use it.  It was probably something that I tried when I couldn't find a different way to obtain the arguments of the vector (probably the myvector.v[1] you mentioned), and never ended up removing it.